decentraland/unity-explorer
# fix: fully removed street view section and logic

## What does this PR change?

Street view has been removed a while ago, but the logic was still inside the code and was downloading resources and consuming memory, this PR removes all related logic to the street view

fixes #2118
...

## How to test the changes?



1. Launch the explorer
2. Check that the satellite view in the map is working as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
